cli_tools/gce_vm_image_export/exporter/exporter.go

```diff
@@ -162,7 +163,7 @@ func Run(clientID string, destinationURI string, sourceImage string, format stri
 	ctx := context.Background()
 	metadataGCE := &compute.MetadataGCE{}
 	storageClient, err := storage.NewStorageClient(
-		ctx, logging.NewStdoutLogger(logPrefix), oauth)
+		ctx, logging.NewStdoutLogger(logPrefix), option.WithCredentialsFile(oauth))
```

Comment:
Was this a bug earlier?

Reply:
The buffered writer has other options to pass in so I changed the param to options instead of passing in the oauth string.